images/scorecard-test-chainsaw/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build the scorecard-test-chainsaw binary
+FROM --platform=$BUILDPLATFORM golang:1.21 as builder
```

Comment:
Id Operator-SDK has the intention to begin to support chainsaw, I think you will need to add an e2e test for those as well. see; https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/go/scorecard_test.go

Reply:
@eddycharly 

Before working on it, I suggest contacting the SDK maintainers in the operator-SDK-dev slack channel to see if they are willing to add it to the project.

Reply:
👍